src/components/PlayButton/PlayButton.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+    ButtonHTMLAttributes,
+    DetailedHTMLProps,
+    HTMLInputTypeAttribute,
+    ReactNode,
+} from "react";
+import { FaPlay, FaPause, FaStop } from "react-icons/fa";
+import style from "./PlayButton.module.scss";
+
+type PlayButtonTypeAttribute = "play" | "pause" | "stop" | "disabled";
+
+type Variant = {
+    icon: ReactNode;
+    colorClass: string;
+};
+
+const playButtonVariants: Record<PlayButtonTypeAttribute, Variant> = {
```

Comment:
Rename to buttonVariants

Reply:
I feel playButtonVariants is more descriptive, why do you want to change the name?

Reply:
Because play is one of the types, you also have pause, stop, disabled